### PR TITLE
Adds CoinFlip commands

### LINKED ISSUE #143: Coin Flip command
`s!coinflip` - Flips a coin
`s!betflip <bet> <heads|tails>` - Flips a coin and bets X switchcoins

### PR DESCRIPTION
**Pull request by bolsomito**
Closes #143

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "api-npm": "^1.0.0",
     "booru": "^0.4.0",
     "bufferutil": "^3.0.5",
+    "cowsay": "^1.3.1",
```
Comment (pedrofracassi) [inline package.json]: y tho

Comment (bolsomito) [reply package.json]: because I motherfucking can

Comment (bolsomito) [reply package.json]: uninstalling it in a sec

Comment (bolsomito) [reply package.json]: fixed

**Comment by pedrofracassi** [comment]
Can we have a custom Switchcoin coin?

**Pull request merged**